src/thunderbots/software/CMakeLists.txt

```diff
@@ -562,6 +562,47 @@ if (CATKIN_ENABLE_TESTING)
             )
     target_link_libraries(gradient_descent_optimizer_test ${catkin_LIBRARIES})
 
+    catkin_add_gtest(navigator_test
+            ai/world/ball.cpp
+            ai/world/field.cpp
+            ai/world/robot.cpp
+            ai/world/game_state.cpp
+            ai/world/team.cpp
+            ai/world/world.cpp
+            ai/intent/catch_intent.cpp
+            ai/intent/chip_intent.cpp
+            ai/intent/direct_velocity_intent.cpp
+            ai/intent/direct_wheels_intent.cpp
+            ai/intent/dribble_intent.cpp
+            ai/intent/intent.cpp
+            ai/intent/kick_intent.cpp
+            ai/intent/move_intent.cpp
+            ai/intent/movespin_intent.cpp
+            ai/intent/pivot_intent.cpp
+            ai/intent/stop_intent.cpp
+            ai/primitive/catch_primitive.cpp
+            ai/primitive/chip_primitive.cpp
+            ai/primitive/direct_velocity_primitive.cpp
+            ai/primitive/direct_wheels_primitive.cpp
+            ai/primitive/dribble_primitive.cpp
+            ai/primitive/kick_primitive.cpp
+            ai/primitive/move_primitive.cpp
+            ai/primitive/movespin_primitive.cpp
+            ai/primitive/pivot_primitive.cpp
+            ai/primitive/primitive.cpp
+            ai/primitive/stop_primitive.cpp
+            ai/navigator/placeholder_navigator/placeholder_navigator.cpp
+            test/ai/navigator/placeholder_navigator/placeholder_navigator.cpp
+            test/ai/navigator/main.cpp
+            test/test_util/test_util.cpp
+            util/parameter/dynamic_parameters.cpp
+            util/timestamp.cpp
```

Comment:
could we glob for this in the interests of keeping the length of the `CMakeLists.txt` down?

Reply:
Sorry, I retract this. We shouldn't be globbing in test executables.